packages/paypal-commerce-integration/src/paypal-commerce/paypal-commerce-payment-initialize-options.ts

```diff
@@ -59,6 +72,11 @@ export default interface PayPalCommercePaymentInitializeOptions {
      */
     onRenderButton?(): void;
 
+    /**
+     * A callback that gets called when strategy is in the process of initialization before rendering Smart Payment Button.
```

Comment:
Also it might be better to add a comment that we return a method to render wallet button as a callback